Finance/teller-track-spending.1h.sh

```diff
@@ -7,7 +7,7 @@
 # - teller-cli: https://github.com/sebinsua/teller-cli#from-release
 # - pcregrep: `brew install pcre`
 #
-# <bitbar.title>teller-track-spending</bitbar.title>
+# <bitbar.title>Track spending at UK banks</bitbar.title>
```

Comment:
I can do that!

I wasn't sure how long it could be to be honest - so I was erring on side of caution.